pkg/env/env_test.go

```diff
@@ -0,0 +1,46 @@
+package env
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import "testing"
+
+func TestEnvironmentType(t *testing.T) {
+	p := &prod{envType: environmentTypeProduction}
+	out := p.IsDevelopment()
+	if out != false {
```

Comment:
Very tiny nit (non-blocking) - these tests are checking the "not" case which I find a bit confusing, seems backwards.